src/ensemble/basic_ensemble_solve.jl

```diff
@@ -1,3 +1,5 @@
+using Logging, ChainRulesCore
```

Comment:
Possibly we need to delete these. idk SciML practices but I don't think the ChainRulesCore one at least is needed

Reply:
Definitely should not be in the file and needs to be at the top level. Also the ChainRulesCore one is not needed. And I think we already import logging?